server/routes/visitJourney/selectVisitors.test.ts

```diff
@@ -752,8 +755,8 @@ testJourneys.forEach(journey => {
     })
 
     it('should save to session and add earliestDate to visitSessionData', () => {
-      const fakeDate = new Date('2023-12-01')
-      jest.useFakeTimers({ advanceTimers: true, now: new Date(fakeDate) })
+      const fakeDateTime = new Date('2023-12-01T09:00')
```

Comment:
Unrelated - fixing subtle issue making test flakey because a time wasn't being specified (and compounded by not resetting timers `afterEach`).